### DISCUSSION TITLE
New OpenOCD,QEMU launch configurations are taking default values from the preferencestore(PersistentPreferences)

### DISCUSSION THREAD

**Discussion by promodkumar-ashling** [eclipse-embed-cdt/eclipse-plugins#562] [Debug Plug-ins Q&A] [answered]
When saving a launch configuration in OpenOCD/QEMU launch the value of each controls are saved to the preference store and when creating a new launch configuration the default values for the controls are taking from the preference store instead of taking it from DefaultPreferences.
Is this considered as a feature in EmbedCDT ? In this case user can't return to default configuration unless changing to a new workspace ?

**Comment by ilg-ul** [comment, marked as answer]
> Is this considered as a feature in EmbedCDT ? 

Yes, CDT does this in other pages, and Embed CDT does the same.

As far as I remember, there is no way to return to the default configuration.
